apps/admin-gui/src/app/shared/components/role-filter-inputs/group-roles-filter.component.html

```diff
@@ -0,0 +1,29 @@
+<mat-form-field class="me-2 fix-select">
+  <mat-label>{{'SHARED.COMPONENTS.GROUP_ROLES_FILTER.FILTER_ROLES' | translate}}</mat-label>
+  <mat-select (selectionChange)="changeRoles()" [formControl]="roles" multiple>
+    <mat-select-trigger>
+      {{displaySelectedRoles() | groupRole}}
```

Comment:
The logic of `displaySelectedRoles` should be inside of the pipe. Only pass in the list of selected roles.
Same goes for the other method + pipe combo.